scripts/grounding/build.sbt

```diff
@@ -0,0 +1,55 @@
+import org.clulab.sbt.BuildUtils
+import org.clulab.sbt.Resolvers
+
+name := "grounding"
+description := "The grounding project implements the org.ml4ai.grounding package including the GroundingApp class."
+
+// Last checked 2021-12-31
+val scala11 = "2.11.12" // up to 2.11.12
+val scala12 = "2.12.15" // up to 2.12.14
+val scala13 = "2.13.7"  // up to 2.13.7
+val scala3  = "3.1.0"   // up to 3.1.0
+
+ThisBuild / crossScalaVersions := Seq(scala12, scala11, scala13, scala3)
+ThisBuild / scalaVersion := crossScalaVersions.value.head
+
+resolvers ++= Seq(
+  ("Artifactory" at "http://artifactory.cs.arizona.edu:8081/artifactory/sbt-release").withAllowInsecureProtocol(true)
+//  Resolvers.localResolver,  // Reserve for Two Six.
```

Comment:
Do we need to keep this commented0out code?

Reply:
No